avm/res/service-fabric/cluster/README.md

```diff
@@ -1056,6 +907,7 @@ module cluster 'br/public:avm/res/service-fabric/cluster:<version>' = {
 
 | Parameter | Type | Description |
 | :-- | :-- | :-- |
+| [`certificate`](#parameter-certificate) | object | Describes the certificate details like thumbprint of the primary certificate, thumbprint of the secondary certificate and the local certificate store location. |
```

Comment:
Please bump a minor version in the version.json file, due to an optional parameter made required

Reply:
Bumped the minor version to 0.2